tokens/src/core/global/transition.json

```
@@ -3,23 +3,47 @@
     "$type": "transition",
     "base": {
       "source": "$transition-base",
-      "$value": "all .2s ease-in-out",
+      "$value": {
+        "property": "all",
+        "duration": ".2s",
+        "timing-function": "ease-in",
```

It looks like this has changed from what was here before

```suggestion
        "timing-function": "ease-in-out",
```

---

Corrected in [(#3222)](https://github.com/openedx/paragon/pull/3222/commits/cf989344f882f25d15ee2c56b0c03cfb1df2a1d9)